assignments/function-conversion.js

```diff
@@ -27,4 +22,4 @@
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
```

Comment:
inside the map you can use the arrow function like so....   .map(num => num * 3)